manifests/pulp_client.pp

```diff
@@ -0,0 +1,48 @@
+# Pulp Client Certs
+class certs::pulp_client (
+  $hostname                          = $::certs::node_fqdn,
+  $generate                          = $::certs::generate,
+  $regenerate                        = $::certs::regenerate,
+  $deploy                            = $::certs::deploy,
+
+  $common_name                       = $::certs::common_name
```

Comment:
I wonder if we shouldn't default `$common_name` to 'admin' here. Given the common_name very much matters for doing client authentication to Pulp, using the default would end up in a broken certificate as I understand things. Using 'admin' at least would default this to the de-facto pulp admin login name and provide a sane certificate if the user didn't pass in a value.